output/output.py

```diff
@@ -0,0 +1,11 @@
+from output.status import Status
+
+
+class OutPut:
```

Comment:
You shouldn't print anything here.  I already explained the concept of API

Reply:
we changed it to print the final message in the main Is this what you mean? if not could you create a meeting with us to understand what you mean? and thanks a lot